src/dimension_slice.c

```diff
@@ -573,6 +573,11 @@ dimension_slice_tuple_found(TupleInfo *ti, void *data)
 	return SCAN_DONE;
 }
 
+/* Scan for a slice by dimension slice id.
+ *
+ * If you're scanning for a tuple you have to provide a lock since otherwise
```

Comment:
If I don't mistake, it is good to place commas:
```suggestion
 * If you're scanning for a tuple, you have to provide a lock, since, otherwise,
```